RELEASE.md

```diff
@@ -168,29 +168,73 @@ point but for now the process is manual.
 1. Open a Powershell terminal in the Windows virtual machine and execute:
 
    ```
-   $ cd c:\users\vagrant\signalfx-agent
-   $ scripts/windows/make.ps1 bundle -AGENT_VERSION "<X.Y.Z>"
+   PS> cd c:\users\vagrant\signalfx-agent
+   PS> .\scripts\windows\make.ps1 bundle -AGENT_VERSION "<X.Y.Z>"
    ```
 
    Where `<X.Y.Z>` is the release version.
 
-1. If the build is successful, verify that
-   `c:\users\vagrant\signalfx-agent\build\SignalFxAgent-X.Y.Z-win64.zip` exists.
+1. If the build is successful, verify that `.\build\SignalFxAgent-X.Y.Z-win64.zip` exists.
 
-1. If everything looks good, run the release script from your local workstation (must be
-   on the Splunk network):
+1. Run the signing script from your local workstation to sign the agent executable in
```

Comment:
The Splunk network? Whatever that means, it should come _before_ the instruction to run the script.